ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/styles.tsx

```diff
@@ -119,15 +119,16 @@ export default {
       maskSize: '100% 100%',
       maskImage: `linear-gradient(
       180deg, ${mobileImageMask})`,
+      aspectRatio: '16 /9',
       [mq.GROUP_4_MIN_WIDTH]: {
+        height: '100%',
+        maxWidth: '60%', // can change this
```

Comment:
Is this the final value for production (does the comment need to be removed?)

Reply:
Will do - more a reminder to self.